doc/source/examples/03_modeling/design_parameters.mystnb

```diff
@@ -0,0 +1,130 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Modeling: Using Design Parameters
+
+You can read and update parameters that are part of the design.
+In this example, we have a simple design with two associated parameters.
+
++++
+
+## Perform required imports
+
+```{code-cell} ipython3
+import os
+from ansys.geometry.core.modeler import *
+from ansys.geometry.core.parameters import *
+```
+
+The file for this example is in the integration tests folder, so let's get its path.
+
+```{code-cell} ipython3
+current_path = os.getcwd()
+file_path = os.path.join(current_path, os.pardir,  os.pardir, os.pardir, os.pardir, "tests/integration/files/blockwithparameters.dsco")
+# Normalize the path to remove any redundant separators
+file_path = os.path.normpath(file_path)
```

Comment:
This is going to fail for any user that tries to download the example and run it. We should perform a local download of this file using the ``requests`` library so that the example is self-contained, rather than directly access the file. Let me know if you need further clarifications. 

This will make the example runnable for anybody.

```suggestion
import requests

def download_file(url, filename):
    response = requests.get(url)
    with open(filename, 'wb') as file:
        file.write(response.content)

url = "https://github.com/..."
file_path = "blockwithparameters.dsco"
download_file(url, filename)
```

Reply:
Adapt the GitHub URL accordingly too

Reply:
@RobPasMue what will be the URL? and how this will be run if the file is not uploaded there yet (it is basically in the branch)? Do I need to install the file first, so the url can work in the notebook run?